### PR TITLE
[5.2] Schema grammar abstraction

### LINKED ISSUE #12419: PostgresGrammar does not check if table exists in schema
In the MySQLGrammer the method compileTableExists checks if the table exists in the configured schema. this is not the case for PostgresGrammer.

I want to use multiple schemas in one Postgresql database with migrate options but that is impossible because it will no create the migrations table if it already exists in one of the schemas.

### PR DESCRIPTION
**Pull request by pelim**
fixes #12419

### CODE REVIEW

File: src/Illuminate/Database/Schema/Grammars/Grammar.php [hunk 1]
```diff
@@ -456,4 +483,5 @@ protected function mapFluentValueToDoctrine($option, $value)
     {
         return $option == 'notnull' ? ! $value : $value;
     }
+
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Schema/Grammars/Grammar.php]: ?

File: src/Illuminate/Database/Schema/Grammars/PostgresGrammar.php [hunk 2]
```diff
@@ -33,13 +33,11 @@ public function compileTableExists()
 
     /**
      * Compile the query to determine the list of columns.
-     *
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Schema/Grammars/PostgresGrammar.php]: please restore that deleted value

**Comment by GrahamCampbell** [comment]
Please send to 5.3.

**Comment by pelim** [comment]
send to 5.3 this cs sytle ci drives me crazy - actually a forget to change my ide cs settings for the laravel repo - can you merge this #13008 in laravel 5.1 because the origin intention of this pr was to fix this error

**Pull request closed without merging**